test/Extensions/TesterAzureUtils/StorageEmulatorUtilities.cs

```diff
@@ -9,7 +8,7 @@ public static class StorageEmulatorUtilities
     {
         public static void EnsureEmulatorIsNotUsed()
         {
-            if (CloudStorageAccount.DevelopmentStorageAccount.ToString().Equals(TestDefaultConfiguration.DataConnectionString, StringComparison.OrdinalIgnoreCase))
+            if (TestDefaultConfiguration.DataConnectionString is { Length: > 0 } connectionString && connectionString.IndexOf("UseDevelopmentStorage", StringComparison.OrdinalIgnoreCase) >= 0)
```

Comment:
The backport uses string.Contains() here. However, string.Contains() is only available in 2.1+ from .NET Framework, but the 3.x branch is on 2.0